### ISSUE TITLE
Bent neck

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9834]
**Preferred term label:**
Bent neck

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustration see WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012866

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
does it make sense to have both [Abnormal sperm neck morphology HP:0012866](https://hpo.jax.org/app/browse/term/HP:0012866) and [Abnormal sperm mid-piece morphology HP:0012867](https://hpo.jax.org/app/browse/term/HP:0012867) or should these terms be merged? If there is a difference, what is it?

**Comment by pnrobinson** [comment]
For this term, I would suggest
label: Bent sperm neck
definition: An abnormal configuration of the neck of spermatazoa such that the neck and tail form an angle of greater than 90% to the long axis of the head. This term should be used if the proportion of spermatazoa with a bent neck is found to be above the normal limit of ???%.

Ideally there would be a PMID for this, but I cannot find it. Alternatively a chapter in the WHO manual.

**Comment by MagoWyr** [comment]
The reference value of normal sperm morphology is that ≥ 4 % of spermatozoa should have normal morphology. For using the term "Bent neck" the WHO does not provide a reference value, but if the value of ≥ 4 % normal shaped spermatozoa is not reached due to bent necks, it should be reported. Therefore we could use this reference value and refer to the WHO lab manual.

**Comment by pnrobinson** [comment]
≥ 4 % of spermatozoa should have normal morphology.
-- really? This would mean that if 95% are abnormal then we would call the finding normal?

**Comment by MagoWyr** [comment]
Yes, that's correct. It is very common that a very high percentage of spermatozoa look abnormal. But if the sperm count is normal (> 39 Million) there will be still enough sperm to fertilise the egg.
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr 
Just to confirm
bent sperm flagella is different from bent neck and the definition could be

The proportion of sperm cells whose flagella is sharply curved or has a sharp angle is above normal limits.

**Comment by MagoWyr** [comment]
Yes, that's different from bent neck and the definition is correct.
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr I do not have access to the WHO book. Could you provide a PMID with a Figure that illustrates this finding and suggest a textual definition?

**Comment by MagoWyr** [comment]
Here it's the same as with the Thin neck:

A PMID for this finding with a description is here: PMID: 22992901

For the figure I would use the one in the WHO laboratory manual: [cid:cbe3d0e8-7c9b-4de7-9cac-0d8313e0a9c0]
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr 
added as

HP:0034959
Label: Bent sperm neck
Def: Sharply angled insertion of the neck and midpiece into the head of a spermatozoon. This term applies if the proportion of spermatozoa with this feature is above the upper limit of normal.

**Comment by MagoWyr** [comment]
Ja, das ist eine gute Beschreibung!
________________________________
